cms/migrations/0027_imagecarouselpage.py

```diff
@@ -0,0 +1,56 @@
+# Generated by Django 2.1.7 on 2019-05-28 11:13
+
+from django.db import migrations, models
+import django.db.models.deletion
+import wagtail.core.fields
+import wagtail.images.blocks
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("wagtailcore", "0041_group_collection_permissions_verbose_name_plural"),
+        ("cms", "0026_text_video_section"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="ImageCarouselPage",
+            fields=[
+                (
+                    "page_ptr",
+                    models.OneToOneField(
+                        auto_created=True,
+                        on_delete=django.db.models.deletion.CASCADE,
+                        parent_link=True,
+                        primary_key=True,
+                        serialize=False,
+                        to="wagtailcore.Page",
+                    ),
+                ),
+                (
+                    "header",
```

Comment:
Since we have moved to wagtail I think we can reuse the `title` attribute from `Page` model on this.